shared/Views/Shared/Components/CourseDetails/_Apply.cshtml

```diff
@@ -10,7 +10,7 @@
 
 @{
   var show2019Apply = FeatureFlags.Apply2019;
-  var showMap = FeatureFlags.Maps;
+  var showMap = ViewBag.showMap;
```

Comment:
Why is one showMap and one previewMap?

Reply:
My reasoning is that `showMap` is what the variable does, its effect. `previewMap` is what we tell our users to append to the URL, and thus communicates to them that they are looking at a preview of an upcoming feature. `?showMap=true` would also work but it wouldn't carry that meaning.